src/pages/products/products.module.scss

```diff
@@ -0,0 +1,22 @@
+@import "normalize.css";
+@import "sanitize.css";
+
```

Comment:
try to include this into global styles. no need to include this imports in each style file

Reply:
this styles included into main.scss and removed from all other modules. All pages look as previously